randomColourModule.js

```diff
@@ -0,0 +1,73 @@
+const body = document.getElementsByTagName('body')[0]
+
+const usedColours = []
```

Comment:
What's the Big-O of searching through an array for pre-existing colors? (.includes) Is there a faster way you could do this same search using another object type? Also, does this variable need to be in the global scope? Is there somewhere else it could go that would make more sense?

Reply:
Again, nice one...! Array searches are O(n) so we can use a Hash table, which is just a simple object in javascript.
Hash tables are O(1).